test/runnable/cppa.d

```diff
@@ -1230,7 +1230,7 @@ void test15455()
 /****************************************/
 // 15372
 
-extern(C++) int foo15372(T)(T v);
+extern(C++) int foo15372(T)(int v);
```

Comment:
Why is this change required?

Reply:
It's not required for `TemplateParameter`, but I figured I'd include it here since it's so trivial. I always make atomic commits, so you can find the explanation in the commit message.